examples/04_example_patchextraction.ipynb

```diff
@@ -36,7 +36,7 @@
         "\n",
```

Comment:
<p>Weird line break here:</p><p> </p><p>Th</p><p>e first line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TIA-Lab/tiatoolbox/pull/132/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>It's probably an issue related to ReviewNB not with the notebook itself. I can see it fine on my notebook locally. Please let me know if there is still an issue.</p>